complete/src/test/groovy/org/openweathermap/OpenweathermapServiceSpec.groovy

```diff
@@ -50,19 +51,31 @@ class OpenweathermapServiceSpec extends Specification implements ServiceUnitTest
         String city = 'London'
         String countryCode = 'uk'
         String appid = 'XXXXX'
-        String expected = '''{"coord":{"lon":-0.13,"lat":51.51},"weather":[{"id":803,"main":"Clouds","description":"broken clouds","icon":"04d"}],"base":"stations","main":{"temp":20.81,"pressure":1017,"humidity":53,"temp_min":19,"temp_max":22},"visibility":10000,"wind":{"speed":3.6,"deg":180,"gust":9.8},"clouds":{"all":75},"dt":1502707800,"sys":{"type":1,"id":5091,"message":0.0029,"country":"GB","sunrise":1502685920,"sunset":1502738622},"id":2643743,"name":"London","cod":200}'''
         ersatz.expectations {
             get('/data/2.5/weather') {
-                query('q',"${city},${countryCode}")
+                query('q', "${city},${countryCode}")
                 query('appid', appid)
                 called(1)
                 responder {
+                    encoder(ContentType.APPLICATION_JSON, Map, Encoders.json) // <1>
                     code(200)
-                    content expected,'application/json'
+                    content([
```

Comment:
Will happily revert this back to the JSON string if you want, but I felt this was cleaner and shows off more of the Ersatz functionality. ;-)